README.md

```diff
@@ -0,0 +1,26 @@
+# metrics-py
```

Comment:
Sure. Just to make it clear, `pluggable-metrics` will be the name `pip install` use? and it'll still be `import metrics` right?